recipes/sdl/all/conanfile.py

```diff
@@ -1,6 +1,11 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.apple import is_apple_os
+from conan.tools.files import get, replace_in_file, rm, rmdir
+from conan.tools.files import patch as apply_patch
```

Comment:
```suggestion
from conan.tools.files import patch
```
Use the original name, it avoids confusion for who is used to read new recipes every day and know most of imports